crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -711,10 +713,10 @@ fn equate(
     match (t, &intel[..]) {
         (&Type::PrimFloat(32), "float") => {}
         (&Type::PrimFloat(64), "double") => {}
-        (&Type::PrimSigned(8), "__int8" | "char") => {}
+        (&Type::PrimSigned(8), "__int8" | "constexpr int" | "char") => {}
         (&Type::PrimSigned(16), "__int16" | "short") => {}
-        (&Type::PrimSigned(32), "__int32" | "const int" | "int") => {}
-        (&Type::PrimSigned(64), "__int64" | "long long") => {}
+        (&Type::PrimSigned(32), "__int32" | "const int" | "int" | "size_t") => {}
```

Comment:
This is not needed, you are using `i64` as `size_t`

Reply:
I will remove that from 32-bit as I do not think amx can be supported on those processors

Reply:
yes, amx is x64-only